src/m3/functions.ts

```diff
@@ -42,7 +42,7 @@ export const isNonDerivedReference = (feature: Feature): feature is Reference =>
 
 
 /**
- * Determines whether a {@link Feature feature} is “relational“,
+ * Determines whether a {@link Feature feature} is “relational”,
```

Comment:
what's about the double quotes at the end?

Reply:
Sorry, my OCD: "this" is different from “this” ;) I've been quite happy since I discovered the keyboard shortcuts for these in macOS...

Reply:
For the sake of simplicity I would use the normal quotes, as in `"this"`